arrow-integrations-jackson-module/src/main/kotlin/arrow/integrations/jackson/module/NonEmptyListModule.kt

```diff
@@ -16,7 +16,7 @@ import com.fasterxml.jackson.databind.type.CollectionType
 import com.fasterxml.jackson.databind.type.TypeFactory
 import com.fasterxml.jackson.databind.util.StdConverter
 
-object NonEmptyListModule : SimpleModule(PackageVersion.VERSION) {
+object NonEmptyListModule : SimpleModule(NonEmptyListModule::class.java.canonicalName, PackageVersion.VERSION) {
```

Comment:
This is now required due to the change in the way module registration detects uniqueness using module name which is done using a string. https://github.com/joansmith/jackson-databind/commit/d672b186bc549227afc210d481fa3dfde197b961